app/src/editor/canvas/state.js

```diff
@@ -1111,3 +1117,51 @@ export function moduleSearch(moduleCategories, search) {
 
     return uniqBy([...exactMatches, ...startsWith, ...nameMatches, ...pathMatches], 'id')
 }
+
+/**
+ * Tries to find a port from one canvas in another canvas.
+ * If failing to match by id, it will match against all of
+ * - the module
+ * - the port name and
+ * - the port type
+ */
+
+function matchPortInOtherCanvas(canvas, prevCanvas, portId) {
```

Comment:
It's trivial but how about renaming this to `matchPortInPreviousCanvas` or something else. "Other" sounds like it could be a subcanvas or another canvas entirely.

Reply:
Sure, I did have it named like this originally actually, will change it

Reply:
Fixed in df78d4a